src/index.js

```diff
@@ -3,8 +3,9 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { HashRouter } from "react-router-dom"
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<HashRouter><App /></HashRouter>, document.getElementById('root'));
```

Comment:
We might want to consider pushing the `HashRouter` into the app directly - if later on we want to do anything with context that tries to modify the router it might be tricky if it sits outside the `App`